src/components/deformation/deformation-model.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 import { inject, observer } from "mobx-react";
 import { BaseComponent } from "../base";
 import { IDisposer, onAction } from "mobx-state-tree";
+import { deg2rad } from "../../utilities/coordinateSpaceConversion";
```

Comment:
`deg2rad()` is imported but never used, while `deg2Rad()` is defined below and used throughout. 🤔

Reply:
well spotted! I wasn't sure if VS code was leading me astray, I guess I shall remove the Rad version!